android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -165,6 +165,7 @@ public class RNCWebViewManager extends SimpleViewManager<WebView> {
   protected @Nullable String mUserAgentWithApplicationName = null;
   protected @Nullable String mDownloadingMessage = null;
   protected @Nullable String mLackPermissionToDownloadMessage = null;
+  protected boolean mAllowsProtectedMedia = false;
```

Comment:
nit: how about moving this up to where `mAllowsFullscreenVideo` is declared?

Reply:
Done :)